### PR TITLE
fix(files): right click actions menu

### PR DESCRIPTION
**Pull request by skjnldsv**
After https://github.com/nextcloud/server/pull/43090

The root changed https://github.com/nextcloud/server/blob/1b0eb2bebd6998b2ebfb8ed177640ea5329bd4ca/apps/files/src/components/FileEntry.vue#L402-L407

### CODE REVIEW

**Review by ShGKme — APPROVED** [review]

File: apps/files/src/components/FileEntry/FileEntryActions.vue [hunk 1]
```diff
@@ -346,7 +346,7 @@ export default Vue.extend({
 <style lang="scss">
 // Allow right click to define the position of the menu
 // only if defined
-.app-content[style*="mouse-pos-x"] .v-popper__popper {
+.content[style*="mouse-pos-x"] .v-popper__popper {
```
Comment (ShGKme) [inline apps/files/src/components/FileEntry/FileEntryActions.vue:349]: Isn't it the same? Do we need root class here at all?

```suggestion
[style*="mouse-pos-x"] .v-popper__popper {
```

**Review by emoral435 — COMMENTED** [review]
Overall good - however, it seems that firefox has had yet another regression (😓😓):

left: chrome | right: firefox
firefox_r3qZP3x53L

I remember this being mentioned in a PR before, but I cannot remember for the life of me which PR it was! If you believe this can be ignored for now / chrome is where most people use NC, I can go ahead and submit an approval

File: apps/files/src/components/FileEntry/FileEntryActions.vue (hunk 1 above)
Comment (ShGKme) [reply apps/files/src/components/FileEntry/FileEntryActions.vue:349]: > Isn't it far less efficient?

Probably it is

**Comment by susnux** [comment]
https://github.com/nextcloud/server/pull/44139

**Pull request closed without merging**